libraries/chem-meta/src/parsing-utils/chemical-table-parser.ts

```diff
@@ -0,0 +1,196 @@
+/* Do not change these import lines to match external modules in webpack configuration */
+import * as grok from 'datagrok-api/grok';
+import * as ui from 'datagrok-api/ui';
+import * as DG from 'datagrok-api/dg';
+
+export interface ChemicalTableParser {
```

Comment:
in Datagrok, I saw interfaces starting from 'I"
so maybe `IChemicalTableParser` would be a better name